Include some headers file, to fix a problem with getrusage and setpriority functions with the new ubuntu raring ringtail development release
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Forwarded: <yes>
Fixed in upstream revision 9223fd2 and 2d0150c
http://boinc.berkeley.edu/trac/changeset/9223fd22e3b6b848cc9e5e151d958542da8fd49d/boinc
http://boinc.berkeley.edu/trac/changeset/2d0150c6872bd3f0489a90a71e2728c3a8e792fb/boinc

--- boinc-7.0.36+dfsg.orig/samples/wrapper/wrapper.cpp
+++ boinc-7.0.36+dfsg/samples/wrapper/wrapper.cpp
@@ -29,4 +29,7 @@
 // Contributor: Andrew J. Younge (ajy4490@umiacs.umd.edu)
 
+#ifndef _WIN32
+#include "config.h"
+#endif
 #include <stdio.h>
 #include <vector>
@@ -36,7 +39,15 @@
 #include "win_util.h"
 #else
+#ifdef HAVE_SYS_WAIT_H
 #include <sys/wait.h>
+#endif
 #include <sys/types.h>
 #include <sys/stat.h>
+#ifdef HAVE_SYS_TIME_H
+#include <sys/time.h>
+#endif
+#ifdef HAVE_SYS_RESOURCE_H
+#include <sys/resource.h>
+#endif
 #include <unistd.h>
 #endif
